lib/sisimai/message/email.rb

```diff
@@ -412,7 +412,7 @@ def self.parse(argvs)
         mesgformat = (mailheader['content-type'] || '').downcase
         ctencoding = (mailheader['content-transfer-encoding'] || '').downcase
 
-        if mesgformat =~ %r{text/(?:plain|html);?}
+        if mesgformat.start_with?('text/plain;') || mesgformat.start_with?('text/html;')
```

Comment:
__[RuboCop]__

Performance/DoubleStartEndWith: Use `mesgformat.start_with?('text/plain;', 'text/html;')` instead of `mesgformat.start_with?('text/plain;') || mesgformat.start_with?('text/html;')`.


<!-- Comment by [SideCI](https://sideci.com) -->